images/kcp-registrar/Dockerfile

```diff
@@ -30,15 +30,16 @@ RUN mkdir /workspace && chmod 777 /workspace && chown 65532:65532 /workspace
 RUN mkdir $HOME && chmod 777 $HOME
 COPY --from=builder workspace/kcp/bin/kubectl-kcp /usr/local/bin/kubectl-kcp
 RUN chmod 755 /usr/local/bin/kubectl-kcp
-COPY ./register.sh /usr/local/bin/register.sh
-RUN chmod 755 /usr/local/bin/register.sh
+RUN microdnf install findutils git
 RUN JQ_VERSION=1.6 && \
-    curl -sSL -o /usr/local/bin/jq https://github.com/stedolan/jq/releases/download/jq-$JQ_VERSION/jq-linux64 && \
+    curl --fail -sSL -o /usr/local/bin/jq https://github.com/stedolan/jq/releases/download/jq-$JQ_VERSION/jq-linux64 && \
     chmod 755 /usr/local/bin/jq
 RUN KUBE_VERSION=v1.24.0 && \
-    curl -L -o /usr/local/bin/kubectl "https://dl.k8s.io/release/$KUBE_VERSION/bin/linux/amd64/kubectl" && \
+    curl --fail -L -o /usr/local/bin/kubectl "https://dl.k8s.io/release/$KUBE_VERSION/bin/linux/amd64/kubectl" && \
     chmod 755 /usr/local/bin/kubectl
+COPY ./register.sh /usr/local/bin/register.sh
+RUN chmod 755 /usr/local/bin/register.sh
```

Comment:
I would expect register.sh to be more stable than jq, kubectl that get regularly released. My feeling is that the change in the ordering will have a negative impact

Reply:
When one does local dev, and iterates on a new feature, the files in the repo are most likely to change. Having to reinstall the application is slowing down the dev cycle. I believe this change is in the best interest of the developer.